internal/service/kafka/kafka.go

```diff
@@ -226,5 +227,12 @@ func resourceKafkaRead(ctx context.Context, d *schema.ResourceData, m interface{
 		})
 	}
 
+	if d.IsNewResource() {
```

Comment:
I think it is up to the user to set this value. What is the reason why we set it to false here?

Reply:
it works incorrectly during import

Reply:
ok ok, now i get it, I think we should make it computed when imported or basically in any other case other then creation.